### PR TITLE
Fix casing in use of Elasticsearch

### PR DESCRIPTION
**Pull request by leifmadsen**
Fix the casing in Elasticsearch from the invalid use of ElasticSearch.
Also fix one minor issue with a missing space.

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/con_node-tuning-operator.adoc [hunk 1]
```diff
@@ -26,29 +26,29 @@
...
+When scheduling an Elasticsearch pod, there must be a label present that matches `tuned.openshift.io/elasticsearch`. If the label is present, one of the two profiles is assigned to the pod. No action is required by the administrator if you use the recommended Operator for Elasticsearch. If you use a custom-deployed Elasticsearch with {ProjectShort}, ensure that you add the `tuned.openshift.io/elasticsearch` label to all scheduled pods.
 
 .Additional resources
 
-* For more information about virtual memory usage by ElasticSearch, see https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html
+* For more information about virtual memory usage by Elasticsearch, see https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/con_node-tuning-operator.adoc:52]:
```suggestion
* For more information about virtual memory use by Elasticsearch, see https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html
```

File: doc-Service-Telemetry-Framework/modules/con_persistent-volumes.adoc [hunk 2]
```diff
@@ -13,4 +13,4 @@ When you enable persistent storage through the Service Telemetry Operator, the P
 
 * For more information about configuring persistent storage for Prometheus in {ProjectShort}, see xref:backends-configuring-persistent-storage-for-prometheus_assembly-installing-the-core-components-of-stf[].
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/con_persistent-volumes.adoc:14]:
```suggestion
* xref:backends-configuring-persistent-storage-for-prometheus_assembly-installing-the-core-components-of-stf[].
```

File: doc-Service-Telemetry-Framework/modules/con_persistent-volumes.adoc [hunk 3]
```diff
@@ -13,4 +13,4 @@ When you enable persistent storage through the Service Telemetry Operator, the P
...
 
-* For more information about configuring persistent storage for ElasticSearch in {ProjectShort}, see xref:backends-configuring-persistent-storage-for-elasticsearch_assembly-installing-the-core-components-of-stf[].
+* For more information about configuring persistent storage for Elasticsearch in {ProjectShort}, see xref:backends-configuring-persistent-storage-for-elasticsearch_assembly-installing-the-core-components-of-stf[].
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/con_persistent-volumes.adoc:16]:
```suggestion
* xref:backends-configuring-persistent-storage-for-elasticsearch_assembly-installing-the-core-components-of-stf[].
```

File: doc-Service-Telemetry-Framework/modules/con_primary-parameters-of-the-servicetelemetry-object.adoc [hunk 4]
```diff
@@ -25,7 +25,7 @@ Support for `servicetelemetry.infra.watch/v1alpha1` was removed from {ProjectSho
 
 Use the `backends` parameter to control which storage back ends are available for storage of metrics and events, and to control the enablement of Smart Gateways that the `clouds` parameter defines. For more information, see xref:clouds_assembly-installing-the-core-components-of-stf[].
 
-Currently, you can use Prometheus as the metrics storage back end and ElasticSearch as the events storage back end.
+Currently, you can use Prometheus as the metrics storage back end and Elasticsearch as the events storage back end.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/con_primary-parameters-of-the-servicetelemetry-object.adoc:28]: See line 20 of this file. Do we still need this important note?

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/con_primary-parameters-of-the-servicetelemetry-object.adoc:28]: No we can remove that for STF 1.5.

**Review by JoanneOFlynn2018 — APPROVED** [review]

**Pull request merged**